src/converter.js

```diff
@@ -44,12 +44,25 @@ function getReferenceName (referenceName, buildingInputType) {
 }
 
 function mapBasicAttributeType (type, attributeName) {
-  switch (type) {
+  let stunt;
+  if (type instanceof Array) {
+    if (type.length === 2) {
+      switch (type[type.length - 1]) {
+        case 'null': throw new Error('type "null" must come first in JSON Schema attribute type array');
```

Comment:
Is it a hard requirement that the null must come first?

Reply:
You're right. It doesn't have to be. I'll change it so that it only depends on whether the type array **contains** 'null'